libsql/internal/ws/websockets.go

```diff
@@ -69,6 +69,15 @@ func (r *execResponse) affectedRowCount() int64 {
 	return int64(r.resp["affected_row_count"].(float64))
 }
 
+func (r *execResponse) lastInsertId() int64 {
+	id, ok := r.resp["last_insert_rowid"].(string)
+	if !ok {
+		return 0
+	}
+	value, _ := strconv.ParseInt(id, 10, 64)
```

Comment:
should we check error here and return 0 in such case?

Reply:
`value` would be zero if there is an error. since we aren't doing anything with the error, I ignored it and returning the `value` as is.